apps/web/src/data/ecosystem.json

```diff
@@ -7,6 +7,30 @@
     "category": "consumer",
     "subcategory": "nft"
   },
+  {
+    "name": "DIMO",
+    "description": "DIMO is connected car platform for developers, enabling user-authorized access to vehicle data to create new types of games, social apps, leaderboards, dashboards, and more. For vehicle owners, the DIMO Mobile app unlocks your car’s data providing valuable vehicle health insights, personalized service offers ,and weekly rewards. DIMO Mobile is your all-in-one app for maximizing your car’s potential and helping you drive smarter.",
```

Comment:
does it need a 'the' in the beginning? DIMO is the connected car platform for developers